### DISCUSSION TITLE
Android 13 permission issue while using offline feature .Not able to download the course throwing error to write file error .

### DISCUSSION THREAD

**Discussion by amantarento** [Sunbird-Ed/Community#724] [Bugs] [answered]
How to fix the permission error in android 13 .

**Comment by swayangjit** [comment, marked as answer]
@amantarento If you only want to read the files then following permissions will work for android 13.

```js 
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>
<uses-permission android:name="android.permission.READ_MEDIA_VIDEO"/>
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>  
```

From android 12 google has restricted all  write access to use the personal space of the device so you can only use the private space of the app.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
